src/clarity-icons/clarity-icons.scss

```diff
@@ -198,7 +198,6 @@ clr-icon {
         top: -1px !important;
 
         overflow: hidden !important;
-        visibility: hidden !important;
```

Comment:
Why are we removing `visibility: hidden`?

Reply:
The main reason was that `visibility: hidden;` makes screen readers ignore the content text. 
https://webaim.org/techniques/css/invisiblecontent/

Reply:
Yep. Totally forgot about that. This is different than popovers.